Marlin/language_tr.h

```diff
@@ -30,213 +30,214 @@
 #ifndef LANGUAGE_TR_H
 #define LANGUAGE_TR_H
 
+//#define SIMULATE_ROMFONT

```

Comment:
> It then makes sense to define SIMULATE_ROMFONT in the non utf version

Okay. That makes sense. So I see we have fonts for the graphical display for Western, Cyrillic, Kana, Greek, Chinese, and Turkish. But these will only work with the UTF-8 versions of the language files.